jetty9/src/main/docker/Dockerfile

```diff
@@ -35,13 +35,17 @@ ENV TMPDIR /tmp/jetty
 ADD jetty-base $JETTY_BASE
 ADD 15-debug-env-jetty.bash 50-jetty.bash /setup-env.d/
 ADD quickstart.sh /scripts/jetty/quickstart.sh
+ADD jetty_start.sh /scripts/jetty/jetty_start.sh
+ADD gen_start_command.sh /scripts/jetty/gen_start_command.sh
 WORKDIR $JETTY_BASE
 RUN mkdir -p webapps $TMPDIR \
  && java -jar $JETTY_HOME/start.jar --add-to-start=setuid \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR \
  && chmod +x /setup-env.d/15-debug-env-jetty.bash \
  && chmod +x /setup-env.d/50-jetty.bash \
- && chmod +x /scripts/jetty/quickstart.sh
+ && chmod +x /scripts/jetty/quickstart.sh \
+ && chmod +x /scripts/jetty/jetty_start.sh \
+ && chmod +x /scripts/jetty/gen_start_command.sh
 
 EXPOSE 8080
-CMD ["java","-Djetty.base=${jetty.base}","-jar","${jetty.home}/start.jar"]
+CMD ["/scripts/jetty/jetty_start.sh"]
```

Comment:
I think it would be better to leave the CMD as the jetty command and then just modify the 50-jetty.bash script to look at  the passed args and if they are for jetty, then replace them with the dry-run output.  Note however, that users should still be able to pass in args like --dry-run or --list-config and have them work correctly.    

I've proposed something similar for the standard docker image: https://github.com/appropriate/docker-jetty/pull/66